frontend/src/scenes/billing/billingProductLogic.ts

```diff
@@ -299,8 +299,10 @@ export const billingProductLogic = kea<billingProductLogicType>([
                         if (props.productRef?.current) {
                             props.productRef?.current.scrollIntoView({
                                 behavior: 'smooth',
-                                block: 'start',
+                                block: 'center',
```

Comment:
I think you can achieve a similar effect without the extra bottom padding. 

Making block: 'start' and upping the timeout delay to even 20 ms will cause the scroll to go all the way to the bottom of the page and fully show the platform and support product card. 

https://github.com/PostHog/posthog/assets/21014901/8953d6da-d255-4c86-9855-551b5aad2c93

Reply:
I want the block to be near the top because otherwise it looks like it's focusing in on the subscribe button in the tile above.